introduction.md

```diff
@@ -1,2 +1,6 @@
 Hi I am Darryl and this is my introduction.
-My favorite code school path is JavaScript.
\ No newline at end of file
+My favorite code school path is JavaScript.
+* JavaScript
+* AngularJS
+* HTML5
+* Sass
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?